src/FormBuilder/resources/js/form-builder/src/settings/group-email-settings/email/template-options/index.tsx

```diff
@@ -0,0 +1,109 @@
+            'give'
+        ),
+        {
+            a: <a href="https://givewp.com/documentation/core/settings/emails/email-tags/" target="_blank" />,
+        }
+    );
+
+    useEffect(() => {
+        if (showPreview) {
+            setShowPreview(false);
+        }
+    }, [notification]);
+
+    return (
+        <>
+            {showPreview && (
+                <>
+                    <EmailPreviewContent emailType={notification} />
+                    <Button
+                        className={'email-preview__back-btn'}
+                        variant={'secondary'}
+                        onClick={() => setShowPreview(false)}
+                    >
+                        {__('Back to template settings', 'give')}
+                    </Button>
+                </>
+            )}
+
+            {!showPreview && (
+                <>
+                    <div className={'email-settings'}>
+                        <EmailTemplateSettings notification={notification} templateTagsRef={templateTagsRef} />
+
+                        {selectedNotificationStatus === 'enabled' && (
+                            <>
+                                <SettingsSection
+                                    title={__('Send a test email', 'give')}
+                                    description={__(
+                                        'Enter the email address you want to send a test email to.',
+                                        'give'
+                                    )}
+                                >
+                                    <PanelRow>
+                                        <SendPreviewEmail
+                                            defaultEmailAddress={emailDefaultAddress}
+                                            emailType={notification}
+                                        />
+                                    </PanelRow>
+                                </SettingsSection>
+
+                                <SettingsSection
+                                    title={__('Template tags', 'give')}
+                                    description={templateTagsDescription}
+                                >
+                                    <PanelRow>
+                                        <ul className={'email-settings-template-tags'} ref={templateTagsRef}>
+                                            {emailTemplateTags.map((tag) => (
+                                                <li key={tag.tag}>
+                                                    <strong>{'{' + tag.tag + '}'}</strong>
+                                                    <p style={{fontSize: '.75rem'}}>{tag.desc}</p>
+                                                    <CopyToClipboardButton text={'{' + tag.tag + '}'} />
+                                                </li>
+                                            ))}
+                                        </ul>
```

Comment:
I think we can swap this out for the template tag component right?

Reply:
Resolved: 32371ee1371370de9e64d9e6813a54017c1040c7